recipes/cad_to_dagmc/meta.yaml

```
@@ -0,0 +1,54 @@
+{% set name = "cad_to_dagmc" %}
+{% set version = "0.6.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b6b7f34dec0d54a19a9476a2c5c157ef23eb3d6796af86dee6798cdc2420fda9
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  noarch: python
```

I notice the Ci is now failing the windows test so perhaps best to put that skip windows back in

```suggestion
  skip: True  # [win]
  noarch: python
```